fortran/CMakeLists.txt

```diff
@@ -12,13 +12,6 @@ endif ()
 #-----------------------------------------------------------------------------
 add_subdirectory (src)
 
-#-----------------------------------------------------------------------------
-# Build the Fortran Examples
-#-----------------------------------------------------------------------------
-if (HDF5_BUILD_EXAMPLES)
-  add_subdirectory (examples)
```

Comment:
Should this PR remove the examples from the fortran subdirectory since they aren't built here anymore, or is this waiting due to Autotools support for the examples being brought in in this PR?

Reply:
The paln is to remove all other example folders once they are incorporated into the HDF5Examples location.